TAS_Engine/Convert/Environment_oM/Space.cs

```diff
@@ -101,6 +101,14 @@ public static BHE.Space ToBHoM(this TBD.zone tbdSpace)
             tasData.Add("Volume", tbdSpace.volume);
             tasData.Add("WallFloorAreaRatio", tbdSpace.wallFloorAreaRatio);
 
+            double[] YearlyPeopleSensibleSepcificGain = Query.GetNumberOfPeople(tbdDocument, tbdSpace);
```

Comment:
It doesn't look like we're using this - can we comment it out for now

Reply:
done